src/components/UI/Alert/Alert.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from "react";
+import { bxIcons } from "../../../data/icons";
+import "./Alert.css";
+
+export const Alert = ({ alert, setAlert }) => {
+	const onClickHandler = () => {
+		setAlert((s) => ({
+			...s,
+			visibility: !s.visibility,
+		}));
+	};
+	const { text, visibility, type } = alert;
+
+	useEffect(() => {
+		const timerVar = setTimeout(() => {
```

Comment:
Suggestion : it provides the timeoutId , give relevant names easy understanding

Reply:
Yeah nice suggestion. Thanks.